deb-get

```diff
@@ -806,6 +786,17 @@ function deb_wraith-master() {
     SUMMARY="An application for controlling the RGB LEDs on AMD's Wraith stock coolers"
 }
 
+function deb_cider() {
+    get_github_releases "https://api.github.com/repos/ciderapp/cider-releases/releases/latest"
+    if [ "${ACTION}" != "prettylist" ]; then
+        URL=$(grep "browser_download_url.*\.deb\"" "${CACHE_DIR}/${APP}.json" | head -n1 | cut -d'"' -f4)
+        VERSION_PUBLISHED="$(echo "${URL}" | cut -d'/' -f8 | sed 's|release-||')"
```

Comment:
```suggestion
        VERSION_PUBLISHED="$(echo "${URL}" | cut -d'/' -f8 | sed 's|^v||')"
```

Reply:
I made the change, but it seems like it's fine either way.

Reply:
I see. I made the change per your request. Thanks for clarifying.

Reply:
for completeness sake, this is the "issue":

The URL that's extract is like this:
`https://github.com/ciderapp/cider-releases/releases/download/v1.5.7/cider_1.5.7_amd64.deb`

Then to extract the version it picks:
The `cut` command in there splits it at the `/`s, (`-d'/'`)and get's the 8th field (`-f8`):
`https:` `<empty>` `github.com`  `ciderapp` `cider-releases` `releases` `download` `v1.5.7` `cider_1.5.7_amd64.deb`
The 8th field being `v1.5.7`

Which to get the actual version number is then fed through to the `sed` command which replaces `^v` (start of string and character `v`) with nothing (empty string), yielding `1.5.7`

The original code that you copied from was trying to remove `release-` (likely because that project was using `release-x.y.z`), so that final step would not break per se, but would leave the `v` in there in front of the version number.